pkg/graph/constructor.go

```diff
@@ -94,3 +96,53 @@ func (g *Graph) AddChannel(channel messagingv1.Channel) {
 
 	v.AddEdge(to, dest, NoTransform)
 }
+
+func (g *Graph) AddTrigger(trigger eventingv1.Trigger) error {
+	brokerRef := &duckv1.KReference{
+		Name:       trigger.Spec.Broker,
+		Namespace:  trigger.Namespace,
+		APIVersion: "eventing.knative.dev/v1",
+		Kind:       "Broker",
+	}
+	brokerDest := &duckv1.Destination{Ref: brokerRef}
+	broker, ok := g.vertices[makeComparableDestination(brokerDest)]
+	if !ok {
```

Comment:
If the broker doesn't exist in the graph, it will say the trigger refers to a non-existent broker. But what if the broker just hasn't been converted to the graph yet? Are there any mechanism to ensure the destination broker is always created first?